allure-generator/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack --optimize-minimize",
     "start": "webpack-dev-server --inline --hot --env.development",
     "pretest": "eslint --ignore-path=.gitignore .",
-    "test": "jest",
+    "test": "jest --colors",
```

Comment:
It breaks console output in Jenkins logs.
It is better not to commit such changes.

Reply:
oh wow, couldn't imagine that can break anythig. For local development is a musthave option, otherwise log is not readable(

there probably shuoul be added some gradlew flag or something like this, i'll research this

Reply:
As Gradle is involved here, I have nothing to say. Usually I ran tests via `npm test` directly out of `allure-generator` folder.